tools/migrate.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# Please ensure the variables in migration vars have been assigned before running this script. 
+
+set -euxo pipefail
+
+(
+# shellcheck source=migration_vars
+source migration_vars
+)
+
+image=${image:-sourcegraph/server}
+sg_dump_file=$(mktemp)
+codeintel_dump_file=$(mktemp)
+
+trap 'rm -f ${sg_dump_file} ${codeintel_dump_file}' EXIT
+
+# Get the  ID of the sourcegraph container
+echo "Obtaing sourcegraph container ID"
+container_id=$(ssh "${src_host}" docker ps | grep "${image}:${version}" | cut -f 1 -d ' ')
+
+# Dump sourcegraph db and alter script for migration
+echo "Dumping sourcegraph database"
+ssh ${src_host} "docker exec ${container_id} pg_dump -C --username=postgres sourcegraph" >${sg_dump_file}
+sed -i '/^CREATE DATABASE/a CREATE USER postgres WITH SUPERUSER CREATEROLE CREATEDB REPLICATION BYPASSRLS;' ${sg_dump_file} >/dev/null 2>&1
+printf '\connect -reuse-previous=on dbname=postgres\nDROP DATABASE sg;\nALTER DATABASE "sourcegraph" RENAME TO sg;\nALTER DATABASE sg OWNER TO sg;\n' >>${sg_dump_file}
+
+# Dump codeintel db and alter script for migration
```

Comment:
cc @slimsag, is codeinsights supported in our `server` image?
Not sure if we need to add migration steps here as well

Reply:
It is not supported. Deploying TimescaleDB inside of single-container would take a fair amount of effort, and given it is for demo purposes only we have made the decision for now to not support it. (a more "minimal" version, without any TimescaleDB backend, might work - though)